### ISSUE TITLE
Menu contrast 

### ISSUE THREAD

**Issue by elizabeth-ux** [hackforla/311-data#1126]
### Overview
"Filters" and "Council" tabs on the query panel need greater contrast to meet tier 2 standards. Most of the text seems okay but I'd still run it past a color contrast checker to make sure since there are a lot of greys used. 

Notes: 
- tab color is #[insert]
- Darker color is #0F181F
- Medium grey tones are often tricky as a backdrop to text.
- Area around the input box for "Address"/"Council" should be #29404F so that it looks like "Date Range", "Request Types" and "Request Status"

## Action Items
- [x] Run color contrast checker to meet tier 2 standards 

## Resources
- https://github.com/hackforla/311-data/issues/1380

**Comment by elizabeth-ux** [comment]
FILTERS word on menu does meet contrast standards for large text. Address/Council and Open/Closed tabs should look like this 
![Frame 22](https://user-images.githubusercontent.com/20293568/134844945-5a700972-8736-482f-88b8-e88e3e730605.png)
with darker color being #0F181F. 
Also, area around input box for Address/Council should be #29404F so that it looks like Date Range, Request Types and Request Status (example 
![Filter bar](https://user-images.githubusercontent.com/20293568/134844686-1b975f34-29f8-434d-a21c-5c6d7db767db.png)

**Comment by elizabeth-ux** [comment]
From the Sept. 30 meeting I attended and in which this issue was discussed, I think this was going to frontend. Not sure if I was supposed to reassign or PMs.

**Comment by EchoProject** [comment]
@melissahermes99 can you discover which colors specifically need to be changed and add that to the requirements above.

**Comment by EchoProject** [comment]
@melissahermes99 I think you completed the task from the comment above. Can you confirm?

**Comment by melissahermes99** [comment]
Yes! This task has been completed on Figma. @nichhk could we double-check that the darker color on the map menu is #0F181F, it's picking up as #192730 for me. Thank you!

**Comment by nichhk** [comment]
Hmm I'm not seeing that color in our codebase. @edwinjue , could you take a look at this?
